.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.35.0
+  - 1.36.0
```

Comment:
What change requires this?

Reply:
I was getting CI failures because the cargo-clippy component couldn't be found. It's possible it was an intermittent failure from Travis, although bumping the Rust version did fix it.

Reply:
I think it was intermittent. Could you drop it?

(in general bumping rust versions is fine, but preferably not as an unrelated change)

Reply:
Agreed, I'll roll it back and hopefully we get a green.